composer.json

```diff
@@ -1,19 +1,13 @@
 {
-    "name": "eko/feedbundle",
+    "name": "muspi/feedbundle",
```

Comment:
Could you keep original bundle name here please? Also restore the `authors` section. Thank you.